src/04_printing.py

```diff
@@ -11,7 +11,16 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
+print('%d' % (x) )
+print('%s' % round(y,2) )
+print('%s' % z)
 
 # Use the 'format' string method to print the same thing
+print('{}'.format(x) )
+print('{0:.2f}'.format(y))
+print('{}'.format(z))
 
-# Finally, print the same thing using an f-string
\ No newline at end of file
+# Finally, print the same thing using an f-string
+print(f"{x}")
+print(f"{y:.2f}" )
+print(f"{z}")
```

Comment:
Good job. We can avoid using the print statement so many times by using commas between each expression. For instance, with the f-string, like so `print(f'x is {x}, y is {y}, z is {z}')`